### PR TITLE
Use an inline wrapper for input fields

### PR DESCRIPTION
**Pull request by Vinnl**
Fixes # #143.

The input field at https://www.cookson-clal.com/login_in.jsp?country=gb&shop=c&language=FRA&site400=001&company=004&siteid=IC is not clickable - our `<div>` wrapper around the input is inside a `<span>` wrapper by the site. As far as I can see there's no need for our wrapper to be a block-level element.

### CODE REVIEW

File: src/js/add_input_icon.js
Comment (maxxcrawford) [inline src/js/add_input_icon.js:126]: Would it be possible to use a custom element (via `customElements.define…`?) and avoid inherited styles? I say that bc this change will absolutely fix Issue #1114, but it may open us up to entirely new website issues who have styles applied to `spans` in their email field/login areas.

**Review by maxxcrawford — APPROVED** [review]
I'm tenativey approving this, but let us hold off on merging it in until we can discuss with the group. Nice fix!

File: src/js/add_input_icon.js
Comment (Vinnl) [reply src/js/add_input_icon.js:126]: Yeah that's definitely something to consider, depending on whether that interferes with host styling on the wrapped input. Additionally, I think we'd want to consider using shadow DOM for at least the Relay icon, to avoid all the `!important`s in our stylesheet.

Comment (maxxcrawford) [reply src/js/add_input_icon.js:126]: Ahhh. I love that it. I'll file that for a seperate enhancement. Great idea!

**Comment by maxxcrawford** [comment]
@groovecoder @lesleyjanenorton I want to get y'alls take on switching out from a `div` to a `span`.

**Comment by groovecoder** [comment]
Whew, I'm out of my depth on this one. I defer to y'all - @maxxcrawford @Vinnl @codemist @lesleyjanenorton .

**Review by groovecoder — APPROVED** [review]

**Comment by Vinnl** [comment]
I'll just close this one - I don't expect it'll fix too many webcompat issues, and the burden of this lying around is bigger than the upside of the change itself :)

**Pull request closed without merging**
